predator/templates/configmap.yaml

```diff
@@ -10,8 +10,12 @@ metadata:
   namespace: {{ .Release.Namespace }}
 data:
   DATABASE_TYPE: {{ .Values.database.type }}
+{{if .Values.database.name}}
   DATABASE_NAME: {{ .Values.database.name }}
```

Comment:
can you please revert this file

Reply:
my mistake. already done during merging master to my branch.